src/functionalTest/resources/data/crown-warned-list/crownWarnedListInvalid.json

```diff
@@ -0,0 +1,65 @@
+{
+  "document": {
+    "publicationDate": "20-02-2020T20:00:00.001Z",
+    "documentName": "This is a document name",
+    "version": "This is a document version"
+  },
+  "venue": {
+    "venueName": "This is the venue name",
+    "venueAddress": {
+      "line": [
+        "Address Line 1"
+      ],
+      "town": "Venue Town",
+      "county": "Venue County",
+      "postCode": "AA1 AA1"
+    }
+  },
+  "courtLists": [
+    {
+      "courtHouse": {
+        "courtRoom": [
+          {
+            "session": [
+              {
+                "sittings": [
+                  {
+                    "sittingStart": "20-02-2020T20:00:00.001Z",
+                    "sittingEnd": "28-02-2020T20:00:00.001Z",
+                    "hearing": [
+                      {
+                        "hearingType": "Directions",
+                        "listNote": "Manual input",
+                        "case": [
+                          {
+                            "caseLinked": [
+                              {
+                                "caseId": "1234",
+                                "name": "A Vs B"
+                              }
+                            ]
+                          }
+                        ],
+                        "party": [
+                          {
+                            "partyRole": "PROSECUTING_AUTHORITY",
```

Comment:
Just wanting to check if PROSECUTING_AUTHORITY should be a party?

It's specifically a reference that's displayed, so potentially prosecuting authority ref instead in the master schema

Reply:
As discussed. Will keep it as it is but will confirm with Iryna and Nigel as this issue seems to affect many other lists